core/store/src/main/java/uk/gov/gchq/gaffer/store/schema/TypeDefinition.java

```diff
@@ -82,7 +82,7 @@ public void setValidateFunctions(final List<Predicate> validateFunctions) {
     /**
      * @return the {@link Serialiser} for the property.
      */
-    @JsonIgnore
+    @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "class")
     public Serialiser getSerialiser() {
```

Comment:
Please can you also update the vertexSerialiserClass getter/setter in Schema.java.